WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/interval/intervaltypes/TypingInterval.java

```diff
@@ -56,10 +76,25 @@ public void run() {
 	 */
 	@SerializedName("diff")
 	JsonifiedLong editDistance;
+	
+	/**
+	 * The number of characters that have been added, removed or modified during
+	 * this interval.
+	 */	
+	@SerializedName("modCount")
```

Comment:
Could you postfix "Diff" to make it apparent that diff, modCountDiff and startEndDiff belong together?